googlemock/CMakeLists.txt

```diff
@@ -37,8 +37,8 @@ endif()
 # as ${gmock_SOURCE_DIR} and to the root binary directory as
 # ${gmock_BINARY_DIR}.
 # Language "C" is required for find_package(Threads).
-project(gmock CXX C)
-cmake_minimum_required(VERSION 2.6.2)
+project(gmock VERSION 1.8.0 LANGUAGES CXX C )
```

Comment:
> I just took this from the original project() call

Okay, I've missed that.

> My cmake knowledge is too limited to decide what is redundant here

As far as I know CMake will check C and C++ compiler by default so there is no need to set these languages explicitly. From what I know you need to set `LANGUAGES C` in cases when C++ compiler is not used or may not exist.

> are the options given somewhere in hunter?

No, Hunter will work fine with any settings.